azure-devops/azext_devops/test/repos/test_policy.py

```diff
@@ -0,0 +1,149 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+from knack.util import CLIError
+try:
+    # Attempt to load mock (works on Python 3.3 and above)
+    from unittest.mock import patch
+except ImportError:
+    # Attempt to load mock (works on Python version below 3.3)
+    from mock import patch
+
+from azext_devops.dev.repos.policy import *
+from azext_devops.dev.common.services import clear_connection_cache
+
+
+class TestUuidMethods(unittest.TestCase):
+
+    _TEST_DEVOPS_ORGANIZATION = 'https://AzureDevOpsCliTest.visualstudio.com'
```

Comment:
Give a bad org name. To avoid missing out on network call. 
We need to find a better way of catching this in CI :)